src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,110 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './TodoList.scss';
+import cn from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
+type Props = {
+  todos: Todo[],
+  setSelectedUserId: (arg0: number) => void,
+  selectedUserId: number,
+};
+
+enum Status {
+  Active = 'Active',
+  Completed = 'Completed',
+  All = 'All',
+}
+
+export const TodoList: React.FC<Props>
+= ({ todos, setSelectedUserId, selectedUserId }) => {
+  const [query, setQuery] = useState('');
+  const [completed, setCompleted] = useState('');
+
+  const isFiltered = () => {
+    if (completed === Status.Active) {
+      return todos.filter(todo => (
+        todo.title.toLowerCase().startsWith(query.toLowerCase())
```

Comment:
the title should include a query, but not just start with a query

need to return todo with title 'spell' if the query is 'ell'